Neos.Flow/Classes/ResourceManagement/EelHelper/StaticResourceHelper.php

```diff
@@ -0,0 +1,80 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Flow\ResourceManagement\EelHelper;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Eel\ProtectedContextAwareInterface;
+use Neos\Flow\ResourceManagement\ResourceManager;
+use Neos\Flow\I18n\Service as I18nService;
+
+class StaticResourceHelper implements ProtectedContextAwareInterface
+{
+    /**
+     * @Flow\Inject
+     * @var ResourceManager
+     */
+    protected $resourceManager;
+
+    /**
+     * @Flow\Inject
+     * @var I18nService
+     */
+    protected $i18nService;
+
+    /**
+     * Get the public uri of a package resource
+     *
+     * @param string $packageKey
+     * @param string $pathAndFilename
+     * @param bool $localize
+     * @return string
+     */
+    public function uri(string $packageKey, string $pathAndFilename, bool $localize = false): string
+    {
+        $resourcePath = sprintf('resource://%s/%s', $packageKey, $pathAndFilename);
+        if ($localize === true) {
+            $localizedResourcePathData = $this->i18nService->getLocalizedFilename($resourcePath);
+            $resourcePath = $localizedResourcePathData[0] ?? $resourcePath;
```

Comment:
Is this intended behavior, to fallback to original? If you ask for a localized version, what do you expect in return? Do we have similar use case of the $localize, where we do the same, so we follow the same pattern everywhere?

Reply:
if a no better version is abvailable the original resource path is used. This is pretty much what the View Helpers and the ResourceUri Fusion Object are doing.

Reply:
Perfect 👍

Reply:
Use case is when you have a special logo for a locale or a special css for languages with right to left fonts